extensions/service_profiles/standard/sections/clause_7_core.adoc

```diff
@@ -4,8 +4,30 @@
 
 include::../requirements/requirements_class_core.adoc[]
 
+=== Profiling Requirements
+
+Profile is conformant with the ModSpec
+
+include::../requirements/core/REQ_modspec.adoc[]
+
+Implementations of the Profile are conformant with EDR Part 1
+
+include::../requirements/core/REQ_EDR-conformant.adoc[]
+
+A common focus of Profiles is to restrict the values of Path parameters. The Profile should fully define requirements for these restrictions. 
+
+include::../requirements/core/REQ_parameter-names.adoc[]
+
+A Profile must be interoperable with "foreign" data providers. Any valid foreign document should be valid under the profile. That means:
+
+* If a data element is valid for EDR, then it should not prohibited under the profile
+* Data elements which are not applicale for the profile domain should be permitted but ignored by processing.
```

Comment:
applicable

Reply:
Also, I am not happy with the word "foreign". I suggest:
`A Profile must be interoperable with other EDR API data providers. Any valid EDR API document should be valid under the profile.

Reply:
Spelling error corrected.  Line 21 changed per the suggestion by @chris-little.

@m-burgoyne restricting profiles to a subset is still an open issue.  We will revisit this once the SWG and OAB make a final decision.